chain/network/src/peer_manager/connection_store/tests.rs

```diff
@@ -0,0 +1,128 @@
+use crate::peer_manager::connection_store::ConnectionStore;
+use crate::peer_manager::connection_store::OUTBOUND_CONNECTIONS_CACHE_SIZE;
+use crate::store;
+use crate::time;
+use crate::types::{ConnectionInfo, PeerInfo};
+use near_crypto::{KeyType, SecretKey};
+use near_primitives::network::PeerId;
+use near_store::NodeStorage;
+use rand::Rng;
+use std::net::{Ipv4Addr, SocketAddr, SocketAddrV4};
+
+fn get_addr(port: u16) -> SocketAddr {
+    SocketAddrV4::new(Ipv4Addr::new(127, 0, 0, 1), port).into()
+}
+
+fn gen_peer_info(port: u16) -> PeerInfo {
+    PeerInfo {
+        id: PeerId::new(SecretKey::from_random(KeyType::ED25519).public_key()),
+        addr: Some(get_addr(port)),
+        account_id: None,
+    }
+}
+
+fn check_outbound_connections(store: &ConnectionStore, wanted: &Vec<ConnectionInfo>) {
+    let got = store.get_recent_outbound_connections();
+    for c in &got {
+        assert!(wanted.contains(c));
+    }
+    for c in wanted {
+        assert!(got.contains(&*c));
+    }
+}
+
+#[test]
+fn test_store_connection() {
```

Comment:
Either document what scenario this test covers, or make the test name more meaningful. Like "test_reload_from_storage".